frontend/src/lib/TriggerUtils.ts

```diff
@@ -310,4 +318,10 @@ export function triggerDisplayString(trigger?: ApiTrigger): string {
     }
   }
   return '-';
+
```

Comment:
IMPORTANT lines 291 to 314 seem to include untranslated text

Reply:
Couldn't find where it is used in the UI